perf_test/sparse/KokkosSparse_sptrsv_superlu.cpp

```diff
@@ -795,19 +795,33 @@ int main(int argc, char **argv) {
     std::cout << "tests[" << i << "] = " << tests[i] << std::endl;
   }
 
-  {
-    using scalar_t = double;
-    //using scalar_t = Kokkos::complex<double>;
-    Kokkos::ScopeGuard kokkosScope (argc, argv);
-    int total_errors = test_sptrsv_perf<scalar_t> (tests, verbose, filename, symm_mode, metis, merge,
-                                                   invert_offdiag, u_in_csr, panel_size, relax_size, loop);
-    if(total_errors == 0)
-      std::cout << "Kokkos::SPTRSV Test: Passed"
-                << std::endl << std::endl;
-    else
-      std::cout << "Kokkos::SPTRSV Test: Failed (" << total_errors << " / " << 2*tests.size() << " failed)"
-                << std::endl << std::endl;
-  }
+  Kokkos::ScopeGuard kokkosScope (argc, argv);
+
+  // If eti-type complex<double> is enabled at compile time, this is what
+  // the perf_test will use
+  #if defined(KOKKOSKERNELS_INST_COMPLEX_DOUBLE)
+    using scalar_t = Kokkos::complex<double>;
+    scalarTypeString = "(scalar_t = Kokkos::complex<double>)";
+  #else
+    // If eti-type double is enabled at compile time, this is what
+    // the perf_test will use
+    #if defined(KOKKOSKERNELS_INST_DOUBLE)
+      using scalar_t = double;
```

Comment:
Double should work fine, right ?

Reply:
Yes, as long as `KOKKOSKERNELS_SCALARS='double'`.